tools/src/command/public_params.rs

```diff
@@ -141,3 +140,12 @@ pub fn sample_test_srs(args: SRSSetupArgs) -> anyhow::Result<PathBuf> {
     gen_test_srs_to_file(num_vars, path_str)?;
     Ok(path)
 }
+
+// TODO: make it accessible to all crates.
```

Comment:
nit: This appears to be stale; consider removing.

```suggestion
```

To verify, I ran `cargo doc --open` from the toplevel directory of this branch and saw that `nexus_tools::command::public_params::format_params_file` is public.

Reply:
this TODO means this function should be somewhere else. Surely you don't want to depend on `nexus-tools` just to be able to format a parameters filename. Maybe it should be part of nova, config, etc. :man_shrugging: